keps/79-hierarchical-cohorts/README.md

```diff
@@ -239,11 +240,27 @@ This is an a core API element and will graduate together with the other core API
 
 * 2023.12.29 - KEP - API and semantics.
 
+## Notes
```

Comment:
This should go inside proposal https://github.com/kubernetes-sigs/kueue/tree/main/keps/NNNN-template#notesconstraintscaveats-optional

Reply:
Done.